### DISCUSSION TITLE
Mars is Terraformed message

### DISCUSSION THREAD

**Discussion by Gugatec** [bafolts/terraforming-mars#5423] [General]
I have been asked by an old-school tfm board game if it is possible to disable the message once mars is terraformed.

As the physical board games don't do that, and we have the option to show VP in realtime, he wants to experience as close as the board game possible.

Does anybody know how to accomplish that?

Cheers.

**Comment by kberg** [comment]
It's not feasible without changing code in the game, I'm sorry.
